src/ic-cdk/src/api/stable.rs

```diff
@@ -107,30 +134,39 @@ pub fn stable_bytes() -> Vec<u8> {
 ///
 /// Will attempt to grow the memory as it writes,
 /// and keep offsets and total capacity.
-pub struct StableWriter {
+pub struct StableWriter<M: StableMemory = CanisterStableMemory> {
```

Comment:
This is not a breaking change. See here - https://doc.rust-lang.org/cargo/reference/semver.html#generic-new-default